### PR TITLE
implemented see all functionality for chat sidebar

### PR DESCRIPTION
**Pull request by Tolik170**
https://github.com/ita-social-projects/SpaceToStudy-Client/assets/63456632/3753dbbf-1f3a-4298-95ee-69ae7c0004d2

image

### CODE REVIEW

File: src/components/sidebar-image-grid/SidebarImageGrid.tsx [hunk 1]
```diff
@@ -31,28 +34,37 @@ const SidebarImageGrid: FC<SidebarImageGridProps> = ({
...
+    const hasMoreElem = index === maxElemToShow - 1 && mediaSize > maxElemToShow
+
+    return (
+      <ClickableImage
+        image={image}
+        key={image._id}
+        onClick={hasMoreElem ? onClick : showImage}
+      >
+        {hasMoreElem && (
+          <Box>
```
Comment (Yurenko) [inline src/components/sidebar-image-grid/SidebarImageGrid.tsx:53]: Would it be more suitable to use an empty fragment instead of a Box?

Comment (Tolik170) [reply src/components/sidebar-image-grid/SidebarImageGrid.tsx:53]: inside ClickableImage component it sets some styles to '& div', so it should be wrapped by Box

**Review by TSlashDreamy — APPROVED** [review]

**Review by Roman-Peretiatko — APPROVED** [review]

**Review by ArturBekhDEV — APPROVED** [review]

File: src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.styles.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const styles = {
+  groupedContent: {
+    container: { p: '24px 0 0 0' },
```
Comment (Yurenko) [inline src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.styles.ts:3]: Do you need this p: '24px 0 0 0'?
Use pt: 24px.

File: src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        >
+          {type === SidebarContentEnum.Media ? (
+            <SidebarImageGrid
+              compactMode={false}
+              images={group.items as Media[]}
+            />
+          ) : (
+            group.items.map((item) => getContentByType(item))
+          )}
+        </AllContentModal>
```
Comment (Yurenko) [inline src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.tsx:51]: Move out these map

File: src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.styles.ts (hunk 2 above)
Comment (Tolik170) [reply src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.styles.ts:3]: by default this container has p: '34px', so I need to rewrite it

File: src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.tsx (hunk 3 above)
Comment (Tolik170) [reply src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.tsx:51]: it need group from groupedItems.map. Also, there is not too much code, so it looks nice

**Review by Yurenko — COMMENTED** [review]
Fix small code smell

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**